cypress.config.js

```diff
@@ -4,5 +4,6 @@ module.exports = defineConfig({
   e2e: {
     setupNodeEvents(on, config) {
     },
+    baseUrl: "https://demoqa.com/automation-practice-form",
```

Comment:
It is better to use "https://demoqa.com/" for baseUrl as we can write tests for other pages